filer/management/commands/generate_thumbnails.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+'''
+Created on May 31, 2016
+
+@author: jakob
+'''
+
+from django.core.management.base import BaseCommand
+from filer.models.imagemodels import Image
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        pks = Image.objects.all().values_list('id', flat=True)
```

Comment:
I read the rationale on the previous PR and I'm ok with this implementation
Would you add a comment on the reason for this syntax?